### PR TITLE
adds Allen Cell Segmenter

### PR DESCRIPTION
**Pull request by neuromusic**
How does this look, @jxchen01 & @benjijamorris?

If you can suggest language for the "Purpose" section, I'm happy to add it in.

### CODE REVIEW

File: projects/allen-cell-segmenter.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Scope
+
+### Key Flows
+
+> Show what the end-to-end experience will be for biologists.
+
+A biologist will be able to
+1. Load some 3D data
+2. Select a workflow which is tuned for a particular cell structure
```
Comment (jxchen01) [inline projects/allen-cell-segmenter.md]: Check the workflow look-up table where each workflow is tuned for a particular structure. Pick the one showing the most similar morphology to user's data

File: projects/allen-cell-segmenter.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+3. Apply the workflow and see results
+4. Tweak parameters for each step in the workflow based on instant feedback from the viewer
+5. Save the parameters + workflow
+6. Apply the parameters + workflow to larger set of images
+7. Analyze the results to extract statistics of cell structures
```
Comment (jxchen01) [inline projects/allen-cell-segmenter.md]: item 7 is out of the scopt of the Alfa Cohort

File: projects/allen-cell-segmenter.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+### Why does this matter to biologists?
+
+### Why does this matter to you and your professional goals and responsibilities?
+
+## Goals
+> What would you like to accomplish during the Alfa Cohort collaboration?
+
+1. Make “lookup table” of segmentation algorithms available. Load workflow & tweak parameters based on instant feedback from the viewer. Target audience: Anyone
+2. [Stretch] For more advanced user who knows a bit about the algorithms, they have the freedom to manipulate the workflow. Freedom to use modules.
+3. [Ultra Stretch] Integrating with ML model. Curation of pass/fail (not painting). Use case: different areas of image may need different algorithm. Then a set of training data is ready for training.
```
Comment (jxchen01) [inline projects/allen-cell-segmenter.md]: I would suggest "Curation of pass/fail (not painting)" --> something like "Curation using the merging or sorting curator (see paper for details)"

File: projects/allen-cell-segmenter.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+layout: page
+---
+
+- *Contributor(s)*: Jianxu Chen (@jxchen01), Benjamin Morris (@benjijamorris)
+- *References*: [Project Site](https://www.allencell.org/segmenter.html), [Github](https://github.com/AllenCell/aics-segmentation), [Publication](https://www.biorxiv.org/content/10.1101/491035v2), [Support](https://forum.allencell.org/c/software-code/segmenter/16)
+
+## Purpose
+
+> Describe the problem you are trying to solve by packaging your tool as a napari plugin in 1-2 sentences.
+I should be able to read this alone and communicate the value to someone else.
```
Comment (jxchen01) [inline projects/allen-cell-segmenter.md:12]: Making a large number of 3D intracelluar structure segmentation algorithms in Allen Cell and Structure Segmenter accesible and easy to tweak for users at any level.

File: projects/allen-cell-segmenter.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
+---
+title: Allen Cell Segmenter
```
Comment (jxchen01) [inline projects/allen-cell-segmenter.md]: we just updated our formal name as "Allen Cell and Structure Segmenter"

Comment (jxchen01) [reply projects/allen-cell-segmenter.md]: I would suggestion somthing similar to this: 

Allen Cell and Structure Segmenter (Segmenter)

Then, anything down below can directly use "Segmenter"

Comment (jxchen01) [reply projects/allen-cell-segmenter.md]: @Kimcmetzler can you comment on this?

File: projects/allen-cell-segmenter.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+- *Contributor(s)*: Jianxu Chen (@jxchen01), Benjamin Morris (@benjijamorris)
+- *References*: [Project Site](https://www.allencell.org/segmenter.html), [Github](https://github.com/AllenCell/aics-segmentation), [Publication](https://www.biorxiv.org/content/10.1101/491035v2), [Support](https://forum.allencell.org/c/software-code/segmenter/16)
+
+## Purpose
+
+> Describe the problem you are trying to solve by packaging your tool as a napari plugin in 1-2 sentences.
+I should be able to read this alone and communicate the value to someone else.
+
+### Why does this matter to biologists?
+
```
Comment (jxchen01) [inline projects/allen-cell-segmenter.md:18]: The large collection of fluorescently-labeled cell lines from the [Allen Cell Collection](https://www.allencell.org/cell-catalog.html) fosters the development of a large set of image processing workflows that serves as a good resource for biologists to use in many different applications.

File: projects/allen-cell-segmenter.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+### Why does this matter to you and your professional goals and responsibilities?
+
```
Comment (jxchen01) [inline projects/allen-cell-segmenter.md:21]: The development of this plugin will enable a broader community of scientists to perform 3D segmentation on their images and aligns with the mission of the [Allen Institute for Cell Science](allencell.org) for open science.

File: projects/index.md [hunk 8]
```diff
@@ -3,7 +3,7 @@ title: Projects
 layout: page
 ---
 
-# Allen Cell Segmenter
+# [Allen Cell Segmenter](projects/allen-cell-segmenter)
```
Comment (jxchen01) [inline projects/index.md]: `[Allen Cell and Structure Segmenter](projects/allen-cell-segmenter)`

**Review by jxchen01 — CHANGES_REQUESTED** [review]

File: projects/allen-cell-segmenter.md (hunk 5 above)
Comment (Kimcmetzler) [reply projects/allen-cell-segmenter.md]: Yes, please use the title, "Allen Cell and Structure Segmenter", and "Segmenter" as the abbreviated title in sections below .

File: projects/allen-cell-segmenter.md (hunk 2 above)
Comment (neuromusic) [reply projects/allen-cell-segmenter.md]: I assume that this is part of their broader workflow, but out of scope for integrating into napari during the Alfa Cohort?

File: projects/allen-cell-segmenter.md (hunk 5 above)
Comment (neuromusic) [reply projects/allen-cell-segmenter.md]: 👍

File: projects/allen-cell-segmenter.md (hunk 2 above)
Comment (neuromusic) [reply projects/allen-cell-segmenter.md]: Added a note to clarify that this would need to happen outside of napari. Does that work?

**Comment by neuromusic** [comment]
OK, made the requested changes. LMK how it looks @jxchen01!

**Review by jxchen01 — APPROVED** [review]
thanks!

**Review by jxchen01 — APPROVED** [review]

**Review by jxchen01 — APPROVED** [review]

**Pull request merged**